.github/container/gcp-tcpx-check.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ -z "${DISABLE_TCPX_CHECK}" ]; then
```

Comment:
```
if [ "${DISABLE_TCPX_CHECK}" == 1 ]; then 
  exit 0
fi
```
may be better since it narrows the scope under the branching condition.

Reply:
Done.